app/assets/stylesheets/modules/_link-unit.scss

```diff
@@ -0,0 +1,10 @@
+.link-unit {
+  &__title {
+    @include core-19;
```

Comment:
There is a `bold-19` mixin.

https://github.com/alphagov/govuk_frontend_toolkit/blob/master/stylesheets/_typography.scss#L133